rpc.opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "thomas@gazagnaire.org"
+authors: "Thomas Gazagnaire, Jon Ludlam"
+homepage: "https://github.com/mirage/ocaml-rpc"
+bug-reports: "https://github.com/mirage/ocaml-rpc/issues"
+dev-repo: "git://github.com/mirage/ocaml-rpc"
+tags: [
+    "org:mirage"
+    "org:xapi-project"
+]
+build-test: [
+    ["jbuilder" "runtest" "-p" name]
+]
+depends: [
+    "jbuilder" {build}
```

Comment:
I can remove core and lwt from here now that rpclib-lwt and rpclib-async are in separate packages

Reply:
I think it's a good idea to remove them.